battery/init.lua

```diff
@@ -152,13 +138,14 @@ widget:buttons(awful.util.table.join(
 -- XXX FIXME
 lib.hooks.timer.register(1, 300, update)
 
-setmetatable(_M, { __call = function ()
+setmetatable(battery, { __call = function ()
   if not backend then
-    backend = backends.get(_M.preferred_backend)
+     backend = backends.get()
```

Comment:
We're going to want to continue using `preferred_backend` so that users can specify a preferred battery backend.

Reply:
Fixed. Thanks.